src/main/scala/io/iohk/ethereum/blockchain/sync/FastSync.scala

```diff
@@ -171,11 +171,13 @@ trait FastSync {
 
     syncStateStorageActor ! fastSyncStateStorage
 
+    private var blockChainOnlyPeers = Set.empty[Peer]
```

Comment:
should we update this Set when peer disconnects and so on? (see: `handlePeerUpdates`)